pkg/api/types.go

```diff
@@ -505,14 +505,20 @@ func (l *VolumeList) Set(value string) error {
 		return errors.New("invalid format, must be source:destination")
 	}
 	var mount []string
-	pos := strings.LastIndex(value, ":")
+	var pos int
+	if strings.HasPrefix(value, "C:\\") {
```

Comment:
don't assume it's the C drive....  or that it's uppercased.

i'm not familiar w/ how mounting works when running on windows but you seem to be stripping the prefix before passing the path to docker which seems like the wrong answer.

@csrwng have you dealt w/ this in cluster up at all?  (passing a windows path as a mount path to a container?)